src/app/(private)/bridge/patients-and-ecmos/map.tsx

```
@@ -0,0 +1,141 @@
+]
+
+const mapOptions = {
+  mapTypeControl: false,
+  streetViewControl: false,
+};
+
+interface MapProps {
+  ecmoList: ECMO[];
+}
+
+const Map: React.FC<MapProps> = ({ ecmoList }) => {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)